server/server.js

```diff
@@ -52,7 +52,8 @@ if (helpers.isTestEnvironment()) {
   server = https.createServer(httpsOptions, app).listen(443)
   setInterval(vitals.checkGatewayHeartbeat, 5 * 60 * 1000)
   setInterval(vitals.checkButtonBatteries, 5 * 60 * 1000)
-  setInterval(vitals.checkButtonHeartbeat, 5 * 60 * 1000)
+  const minutesBetweenHeartbeatChecks = parseInt(helpers.getEnvVar('VITALS_MINUTES_BETWEEN_HEARTBEAT_CHECKS'), 10)
+  setInterval(vitals.checkButtonHeartbeat, minutesBetweenHeartbeatChecks * 60 * 1000)
```

Comment:
There are two different kinds of heartbeat checks here, Buttons and Gateways. I know that you've merged this already, but could you please rename the environment variable to make it obvious which one this is?

Thanks @micahdbak !